carlsim/test/compartments.cpp

```diff
@@ -174,6 +174,7 @@ TEST(COMPARTMENTS, spikeTimesCPUvsData) {
 *
 * This test makes sure that CPU mode and GPU mode of the compartment model produce the exact same spike times,
 * even with RK4 at 50 time steps.
+* FIX ME: TEST CURRENTLY FAILS AT numIntSteps = 30 (or 40 depending on machine) with small spike time deviation (2ms)
```

Comment:
I was going to work with Hirak on Neuron Monitor on Wednesday, but I suppose we could do this instead. I agree that this bug should be fixed, but I would not consider it to be feature breaking. I was unable to replicate any deviations in debug mode. And even in release mode it is only a 2ms deviation between CPU & GPU spiketimes at a single timestep. I'll update you on the progress.